internal/rollbar/rollbar.go

```diff
@@ -91,6 +90,12 @@ func SetupRollbar(token string) {
 		// We're not a server, so don't send server info (could contain sensitive info, like hostname)
 		data["server"] = map[string]interface{}{}
 		data["platform_os"] = runtime.GOOS
+		data["request"] = map[string]string{
+			// Rollbar specially interprets the body["request"]["user_ip"] key, but it does not have to
+			// actually be an IP address. Use device_id for now. We may want to use real IP later if it
+			// is easily accessible (Rollbar does not provide it for us with non-http.Request errors).
+			"user_ip": uniqid.Text(),
+		}
```

Comment:
I'm guessing `data["request"]` isn't normally set at this time? Can we just add a check to instantiate it if it's not set, and then set the user_ip.

I'm concerned there might be scenario's where we would override other request parameters.

Reply:
Correct. It's only set for an error involving an `http.Request`. I've added the check just in case though.